File: raviproducts/raviproducts/report/custom_sales_invoice_trends/custom_sales_invoice_trends.py
# Copyright (c) 2022, VPS Consultancy and contributors
# For license information, please see license.txt

import frappe

# from erpnext.controllers.trends import 
from raviproducts.custom_trend import get_data,get_columns

def execute(filters=None):
	if not filters:
		filters = {}
	data = []
	conditions = get_columns(filters, "Sales Invoice")
	data = get_data(filters, conditions)

	return conditions["columns"], data

File: raviproducts/raviproducts/report/custom_sales_invoice_trends/custom_sales_invoice_trends.js
// Copyright (c) 2022, VPS Consultancy and contributors
// For license information, please see license.txt
/* eslint-disable */

// frappe.query_reports["Custom Sales invoice trends"] = {
// 	"filters": [

// 	]
// };


frappe.require("assets/erpnext/js/sales_trends_filters.js", function() {
	frappe.query_reports["Custom Sales invoice trends"] = {
		filters: erpnext.get_sales_trends_filters()
	}
});

